Modules/Core/Common/include/itkPointSet.hxx

```diff
@@ -66,6 +66,41 @@ PointSet<TPixelType, VDimension, TMeshTraits>::SetPoints(PointsContainer * point
   }
 }
 
+template <typename TPixelType, unsigned int VDimension, typename TMeshTraits>
+void
+PointSet<TPixelType, VDimension, TMeshTraits>::SetPointsUsingArray(PointsVectorContainer * points)
+{
+  itkDebugMacro("setting Points container to " << points);
+
+  if (points->Size() % PointDimension != 0)
+  {
+    itkExceptionMacro("Number of entries in given 1d array incompatible with the point dimension");
+  }
+
+  unsigned long int numberOfPoints = points->Size() / PointDimension;
```

Comment:
`unsigned long int` -> `SizeValueType` (`unsigned long int` is only 32 bits on Windows).

Reply:
used IdentifierType where appropriate.